pythonAPIClient/client.py

```diff
@@ -296,13 +296,170 @@ def usage_by_client(self, clientID, owner):
+                city : str
+                    City which the camera locates at.
+                resolution_width : num
+                    Resolution width of the camera.
+                resolution_height : num
+                    Resolution height of the camera.
+                utc_offset : num
+                    Time difference between UTC and the camera location.
+                timezone_id : str
+                    Time zone ID of the camera location.
+                timezone_name : str
+                    Time zone name of the camera location.
+                reference_logo : str
+                    Reference logo of the camera.
+                reference_url : str
+                    Reference url of the camera.
+                port : str
+                    (ip_camera) Port to connect to camera.
+                brand : str
+                    (ip_camera) Brand of the camera.
+                model : str
+                    (ip_camera) Model of the camera.
+                image_path : str
+                    (ip_camera) Path to retrieve images from the camera.
+                    if the camera is an ip camera and 'is_active_image' is true,
+                    then it will always have a image_path.
+                    However, image_path can exist even if 'is_active_image'
+                    is false for this ip camera.
+                video_path : str
+                    (ip_camera) Path to retrieve video from the camera.
+                    if the camera is an ip camera and 'is_active_video' is true,
+                    then it will always have a video_path.
+                    However, video_path can exist even if 'is_active_video'
+                    is false for this ip camera.
+
+                Raises
+                ------
+                    AuthenticationError
+                        If the client secret of this client object does not match the clientID.
+                    FormatError
+                        List of invalid attributes.
+                    ResourceConflictError
+                        The legacy_cameraID already exist in the database.
+                    InternalError
+                        If there is an API internal error.
+                    ResourceNotFoundError
+                        If no client app exists with the clientID of this client object.
+
+                Returns
+                -------
+                str
+                    cameraID: the new camera ID for the created camera
+        """
+
+        local_params = dict(locals())
+
+        if self.token is None:
+            self.request_token()
+
+        url = Client.base_URL + 'cameras/create'
+
+        local_params['type'] = local_params.pop('camera_type')
+        del local_params['self']
+
+        if is_active_video is None:
+            raise FormatError('Must provide is_active_video')
+        if is_active_image is None:
+            raise FormatError('Must provide is_active_image')
```

Comment:
I think we can remove manually checking if required fields exists in the python client. for the reason that 
1. it will be returned by the API anyway. and you will be able to return a format error
2. when there are multiple format errors, for example, missing camera_type value, missing is_active_video, and a is_active_image='is_active' in one request will result in error message spotting all three mistakes in one error message, which we could not do in the python client.